docs/user/sbt.rst

```diff
@@ -107,7 +107,7 @@ Garbage collectors
    for short-running command-line applications or applications where garbage
    collections pauses are not acceptable.
 
-3. **immix.** (experimental, since 0.3)
+3. **immix.** (default, since 0.3)
```

Comment:
Lets reorder this list to make immix come first, boehm second and none last. None is still experimental.

Reply:
Sounds good.